src/Controller/EventNodeController.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * @file
+ */
+
+namespace Drupal\popapi\Controller;
+
+use Drupal\node\Entity\Node;
+
+/**
+ * Class EventNodeController
+ * @package Drupal\popapi\Controller
+ */
+class EventNodeController {
+
+  /**
+   * @return array|int
+   */
+  public static function loadLatestEventDate() {
+    $query = \Drupal::entityQuery('node');
+    $query->condition('status', 1);
```

Comment:
drupal has a defined variable for the "1" called "NODE_PUBLISHED". see core/modules/node/node.module